root/edit/details/AddMedium.js

```diff
@@ -73,7 +73,7 @@ const CondensedTrackACs = ({
   }
 
   let thisCredit;
-  let thisPosition = 0;
+  let thisPosition = tracks[0].position - 1;
```

Comment:
Starting at thisPosition = 0 and adding 1 before comparison broke for stuff where the position *is* supposed to be 0. Now it should be safe.

Reply:
Will that continue to work once mediums can have an empty tracklist? We bail out if `!tracks`, but not if `!tracks?.length`.

Reply:
We shouldn't get there once https://github.com/metabrainz/musicbrainz-server/pull/1633 gets merged AFAICT.